instrumentation/quartz-2.0/library/src/main/java/io/opentelemetry/instrumentation/quartz/v2_0/TracingJobListener.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.quartz.v2_0;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.quartz.JobListener;
+import org.quartz.SchedulerException;
+
+final class TracingJobListener implements JobListener {
+
+  private final Instrumenter<JobExecutionContext, Void> instrumenter;
+
+  TracingJobListener(Instrumenter<JobExecutionContext, Void> instrumenter) {
+    this.instrumenter = instrumenter;
+  }
+
+  @Override
+  public String getName() {
+    return TracingJobListener.class.getName();
+  }
+
+  @Override
+  public void jobExecutionVetoed(JobExecutionContext jobExecutionContext) {
+    // TODO(anuraaga): Consider adding an attribute for vetoed jobs.
```

Comment:
Is a veto similar to cancelling? We have some `<someting>.cancelled` attributes being added in various instrumentations.

Reply:
Let me leave this as a todo - I think cancelled is about right but lacking a semantic convention and significant knowledge of quartz will stick with simple for now.